### ISSUE TITLE
Metro Bundler not start with react native 0.73.0 when run react-native run-android

### ISSUE THREAD

**Issue by MarcosGomesDev** [facebook/react-native#41867]
### Description

I created an app with react native 0.73, when i run react-native run-android my metro bundler not open but my terminal install app android when started app i recetive this message "Unable to load script make sure you are either running a Metro server", its working only run react-native start and press A for android

### Steps to reproduce

clone repository
npm install
npm run android

### React Native Version

0.73.0

### Affected Platforms

Runtime - Android

### Output of `npx react-native info`

```text
System:
  OS: Windows 10 10.0.19045
  CPU: (8) x64 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz
  Memory: 4.47 GB / 15.73 GB
Binaries:
  Node:
    version: 20.9.0
    path: C:\Program Files\nodejs\node.EXE
  Yarn:
    version: 1.22.21
    path: ~\AppData\Roaming\npm\yarn.CMD
  npm:
    version: 10.2.4
    path: C:\Program Files\nodejs\npm.CMD
  Watchman: Not Found
SDKs:
  Android SDK:
    API Levels:
      - "29"
      - "33"
      - "34"
    Build Tools:
      - 29.0.0
      - 30.0.3
      - 33.0.0
      - 33.0.1
      - 34.0.0
    System Images:
      - android-29 | Google APIs Intel x86 Atom
      - android-33 | Google APIs Intel x86_64 Atom
      - android-34 | Google APIs Intel x86_64 Atom
    Android NDK: Not Found
  Windows SDK: Not Found
IDEs:
  Android Studio: AI-223.8836.35.2231.11005911
  Visual Studio:
    - 16.11.34301.259 (Visual Studio Community 2019)
Languages:
  Java: 17.0.9
  Ruby: Not Found
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.73.0
    wanted: 0.72.3
  react-native-windows: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: true
  newArchEnabled: false
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found
```

### Stacktrace or Logs

```text
dont have, just showing message on emulator android or physical device
```

### Reproducer

https://github.com/MarcosGomesDev/ErrorReactNative

### Screenshots and Videos

![image](https://github.com/facebook/react-native/assets/69877839/f909320d-f0a9-4675-a8c3-e3603bf8fb5f)
![image](https://github.com/facebook/react-native/assets/69877839/5b0fbd18-fed7-47f0-a6ae-be863401a14a)

**Comment by erez-unitronics** [comment]
Same for me

**Comment by Mehdi-KHALFALLAH** [comment]
same here, is it now required to run the metro bundler manually?

**Comment by JavoxirNazarov** [comment]
same

**Comment by james-cohen** [comment]
Same. Unfortunately using react-native start isn't sufficient because I can't pass Android or iOS specific parameters. Having to run metro manually in one terminal and then open another to run the platform specific command isn't ideal

**Comment by saoudnazir** [comment]
same issue

**Comment by fabriziobertoglio1987** [comment]
There is already a PR that fixes this issue https://github.com/react-native-community/cli/pull/2198

**Comment by city0666** [comment]
Same for me

**Comment by lyswhut** [comment]
I try to edit `node_modules/@react-native-community/cli-tools/build/findDevServerPort.js` file:
```diff
- let startPackager = false;
+ let startPackager = true;
```
Then I got:
```
> react-native run-android --active-arch-only
error Cannot start server in new window because no terminal app was specified.
```

**Comment by truongqt** [comment]
same here. I have to open terminal and run `yarn start` manually, then reloading to work

**Comment by cortinico** [comment]
Locking as folks are commenting "same issue" while we mentioned that this issue is already resolved and will be include in 0.73.1

**Comment by lunaleaps** [comment]
I actually think this wasn't picked in 0.73.1, as it looks like the fix is in [v13.0.1](https://github.com/react-native-community/cli/releases/tag/v13.0.1). It will be picked into 0.73.2

**Comment by cortinico** [comment]
Closing as 0.73.2 was released yesterday
